golang/cosmos/x/vbank/vbank_test.go

```diff
@@ -748,10 +748,18 @@ func Test_EndBlock_Rewards(t *testing.T) {
 
 type mockAuthKeeper struct{
 	accounts map[string]authtypes.AccountI
+	modAddrs map[string]string
 }
 
-func (ma mockAuthKeeper) GetModuleAddress(name string) sdk.AccAddress {
-	return sdk.AccAddress(name)
+func (ma mockAuthKeeper) GetModuleAccount(ctx sdk.Context, name string) authtypes.ModuleAccountI {
+	if addr, ok := ma.modAddrs[name]; ok {
+		acct, ok := ma.accounts[addr]
+		if !ok {
+			panic("missing module account")
+		}
+		return acct.(authtypes.ModuleAccountI)
+	}
+	return nil
```

Comment:
Consider putting the successful case on the "spine" with error exits on indented "ribs".

```suggestion
	addr, ok := ma.modAddrs[name]
	if !ok {
		return nil
	}
	acct, ok := ma.accounts[addr]
	if !ok {
		panic("missing module account")
	}
	return acct.(authtypes.ModuleAccountI)
```

Or maybe go all-in on the compound if-conditions for a minimal line count. It's test code, so have fun!
```suggestion
	if addr, ok := ma.modAddrs[name]; ok {
		if acct, ok := ma.accounts[addr]; ok {
			return acct.(authtypes.ModuleAccountI)
		}
		panic("missing module account")
	}
	return nil
```

Reply:
Hmm.  Choices, choices!  I prefer your first suggestion.